.github/workflows/release-beta.yml

```
@@ -0,0 +1,60 @@
+
+      - name: Run Build
+        if: env.ABORT != 'true'
+        run: pnpm build && pnpm build:taro
+
+      - name: Run Release @nutui/nutui-react
+        if: env.ABORT != 'true'
+        run: cd ./release/h5 && npm publish --dry-run --tag beta
+
+      - name: Run Releases @nutui/nutui-react-taro
+        if: env.ABORT != 'true'
+        run: cd ./release/taro && npm publish --dry-run --tag beta
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)